src/site/fml/faq.fml

```diff
@@ -301,11 +301,11 @@ mvn deploy:deploy-file \
         </p>
       </answer>
     </faq>
-    <faq id="On Windows with Sun JDK, why javadoc:javadoc goal blows up due to an IllegalArgumentException in sun.net.www.ParseUtil.decode()">
-      <question>On Windows with Sun JDK, why <i>javadoc:javadoc</i> goal blows up due to an IllegalArgumentException in <i>sun.net.www.ParseUtil.decode()</i>?</question>
+    <faq id="On Windows with Oracle JDK, why javadoc:javadoc goal blows up due to an IllegalArgumentException in sun.net.www.ParseUtil.decode()">
+      <question>On Windows with Oracle JDK, why <i>javadoc:javadoc</i> goal blows up due to an IllegalArgumentException in <i>sun.net.www.ParseUtil.decode()</i>?</question>
       <answer>
         <p>
-          You are on Windows XP with Sun JDK 5 or JDK 6 freshly installed, and when you run <i>mvn javadoc:javadoc</i>,
+          You are on Windows XP with Oracle JDK 5 or JDK 6 freshly installed, and when you run <i>mvn javadoc:javadoc</i>,
```

Comment:
I'm tempted to delete this one entirely since both the Windows and JDK referenced are so old. However, I don't know if this might happen in a nee system or not.

Reply:
Delete it. Time to get over it.

Reply:
done